cmd/root/user_agent_command.go

```diff
@@ -33,5 +34,11 @@ func commandString(cmd *cobra.Command) string {
 }
 
 func withCommandInUserAgent(ctx context.Context, cmd *cobra.Command) context.Context {
-	return useragent.InContext(ctx, "cmd", commandString(cmd))
+	// A UUID that'll will allow use to correlate multiple API requests made by
+	// the same command invocation.
+	// When we add telemetry to the CLI, this trace ID will allow allow us to
+	// correlate logs in HTTP access logs with logs in Frontend logs.
+	newCtx := useragent.InContext(ctx, "command-trace-id", uuid.New().String())
+
+	return useragent.InContext(newCtx, "cmd", commandString(cmd))
 }
```

Comment:
Please move to a separate file + function.

We have precedent for the command and the upstream parameters.